Makefile

```diff
@@ -31,7 +31,7 @@ dockertestimage:
 	docker build --build-arg GDAL=$(GDAL) --target gdal -t rasterio:$(GDAL) .
 
 dockertest: dockertestimage
-	docker run -it -v $(shell pwd):/app --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --entrypoint=/bin/bash rasterio:$(GDAL) -c '/venv/bin/python setup.py develop && /venv/bin/python -B -m pytest -rP -m "not wheel" --cov rasterio --cov-report term-missing $(OPTS)'
+	docker run -it -v $(shell pwd):/app --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY --entrypoint=/bin/bash rasterio:$(GDAL) -c '/venv/bin/python setup.py develop && /venv/bin/python -B -m pytest -m "not wheel" --cov rasterio --cov-report term-missing $(OPTS)'
```

Comment:
I accidentally turned up the reporting in a previous commit and this rolls it back.